code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -411,7 +411,7 @@
 			if(show_message)
 				to_chat(M, "<span class='danger'>You feel your burns and bruises healing! It stings like hell!</span>")
 			SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "painful_medicine", /datum/mood_event/painful_medicine)
-			//Has to be at less than THRESHOLD_UNHUSK burn damage and have 100 isntabitaluri before unhusking. Corpses dont metabolize.
+			//Has to be at less than THRESHOLD_UNHUSK burn damage and have 100 instabiluri (WHAT) before unhusking. Corpses dont metabolize.
```

Comment:
```suggestion
			//Has to be at less than THRESHOLD_UNHUSK burn damage and have 100 instability before unhusking. Corpses dont metabolize.
```
I'm not entirely sure how they managed that either.

Reply:
Apparently this is the term they use for... whatever it is. 
https://github.com/tgstation/tgstation/pull/48717

Reply:
![](https://file.house/C2lp.png)
It's their term for synthflesh, apparently. It doesn't apply here at least.